build.gradle

```diff
@@ -3,18 +3,16 @@
 buildscript {
     repositories {
         google()
-        jcenter()
         mavenCentral()
     }
 
     ext.versions = [
-            'kotlin': '1.3.31',
+            'kotlin': '1.3.40',
```

Comment:
I realise the point of this PR isn't to update to the latest libs for everything but it would be nice if we're able to bump to the latest minor version if we're doing any update, e.g. 1.3.72?

Reply:
Done